tools/fileinfo/features/pe-loader-corruptions/test.py

```diff
@@ -439,7 +439,8 @@ class Test039(Test):
         tool='fileinfo',
         input=[
             '039-import-thunk-invalid-32bit.ex_',
-            '039-import-thunk-invalid-64bit.ex_'
+            '039-import-thunk-invalid-64bit.ex_',
+            '039-import-thunk-invalid-name-64bit.ex_'
```

Comment:
It seems that you have forgotton to include the newly tested file (`039-import-thunk-invalid-name-64bit.ex_`) as `retdec-fileinfo` fails with `The input file probably not exist or it is incorrect`. Could you please remedy that?

Reply:
Fixed in commit d6ce4f98ca2d0e47ad8adb2ae2f935d607162b79